src/lib/src/header/header.component.css

```diff
@@ -124,76 +61,6 @@ nav.alternate {
   }
   h1 a {
     line-height: 73px;
-    -webkit-background-size: auto 33px;
-    -moz-background-size: auto 33px;
     background-size: auto 33px;
   }
-  a.drawer {
-    position: absolute;
-    left: -10px;
-    top: 12px;
-  }
-  nav {
-    width: 50%;
-  }
-  nav .nav-holder {
-    height: 73px;
-    top: -29.5px;
-  }
-  nav .nav-holder > *:not(.dropdown) {
-    line-height: 73px;
-  }
-}
-
-/*
- * Loading indicator
- */
-header.loading {
```

Comment:
not used, the loading indicator is on the user image

Reply:
not actually an issue, but i notice the spinner isn't horizontally-centered on the image:

![image](http://cav.is/i/9mm30.gif)

Reply:
this positioning is fudged into place with negative margins and is an absolute nightmare :(